shell/platform/android/ndk_helpers.cc

```diff
@@ -22,6 +22,9 @@ typedef void (*fp_AHardwareBuffer_acquire)(AHardwareBuffer* buffer);
 typedef void (*fp_AHardwareBuffer_release)(AHardwareBuffer* buffer);
 typedef void (*fp_AHardwareBuffer_describe)(AHardwareBuffer* buffer,
                                             AHardwareBuffer_Desc* desc);
+typedef void (*fp_AHardwareBuffer_getId)(AHardwareBuffer* buffer,
```

Comment:
I think returning the uint64_t is a more natural API, WDYT?

Reply:
Done